src/main/java/nl/novi/bachwtechiteasy/dtos/UserDto.java

```diff
@@ -0,0 +1,16 @@
+package nl.novi.bachwtechiteasy.dtos;
+
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import nl.novi.bachwtechiteasy.models.Authority;
+
+import java.util.Set;
+
+public class UserDto {
+    public String username;
+    public String password;
+    public Boolean enabled;
+    public String apikey;
+    public String email;
+    @JsonSerialize
+    public Set<Authority> authorities;
```

Comment:
Beter om geen Entities in je DTO's te gebruiken. Deze is erg simpel (heeft geen verder relaties), maar alsnog. Enkel een `Set<String>` (met de namen van de rollen) was hier ook goed geweest